examples/Makefile-eagle

```diff
@@ -0,0 +1,258 @@
+#
```

Comment:
Not necessary to be addressed in this PR, please consider supporting `CMake` too.